### PR TITLE
create pipe

### PR DESCRIPTION
**Pull request by ervand96**

### CODE REVIEW

File: src/app/pipe/custop.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+export class PricePipe implements PipeTransform {
+  transform(
+    value: number,
+    pipeType: string,
+    result: string = 'more' || 'less',
+    numbers: number
+  ) {
+    if (result === 'more') {
+      value = value / 1000;
+      return pipeType + value.toFixed(numbers);
```
Comment (KarenChilingaryan) [inline src/app/pipe/custop.pipe.ts]: if you use toFixed you can get 0 (for example: 20.32.toFixed(4) = 20.3200 )
please use other version

File: src/app/pipe/custop.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    } else {
+      return pipeType + value.toFixed(numbers);
```
Comment (KarenChilingaryan) [inline src/app/pipe/custop.pipe.ts]: the same here for toFixed

File: src/app/pipe/custop.pipe.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+})
+export class PricePipe implements PipeTransform {
+  transform(
+    value: number,
+    pipeType: string,
+    result: string = 'more' || 'less',
+    numbers: number
+  ) {
+    if (result === 'more') {
+      value = value / 1000;
```
Comment (KarenChilingaryan) [inline src/app/pipe/custop.pipe.ts]: don't change argument value

File: src/app/pipe/custop.pipe.ts [hunk 4]
```diff
@@ -11,10 +11,10 @@ export class PricePipe implements PipeTransform {
     numbers: number
   ) {
     if (result === 'more') {
-      value = value / 1000;
-      return pipeType + value.toFixed(numbers);
+      const check = value / 1000;
```
Comment (Karen-Avdalyan) [inline src/app/pipe/custop.pipe.ts]: why it's called `check`?

**Pull request merged**